pcap.c

```diff
@@ -2104,22 +2109,21 @@ pcap_offline_filter(const struct bpf_program *fp, const struct pcap_pkthdr *h,
  * from the library but being truncated).
  */
 #ifdef HAVE_VERSION_H
-#include "version.h"
-#else
-static const char pcap_version_string[] = "libpcap version 1.x.y";
+#include "..\..\version.h"
```

Comment:
I found that it's impossible for me to put ``version.h`` into the same folder as the source code, because currently libpcap is a submodule of Npcap. And the Npcap repo won't track any files inside the submodule folder, so Npcap repo can't track ``versions.h``.

I think I will use ``_WIN32`` macro to differentiate the Windows condition.